### PR TITLE
[Editor] Add a first test to test the new alt text flow

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

**Comment by timvandermeij** [comment]
Good to see test coverage for the new flow! I'll review this either tomorrow or during the weekend.

**Review by timvandermeij — COMMENTED** [review]
Looks good to me, with a few smaller comments.

File: test/integration/stamp_editor_spec.mjs [hunk 1]
```diff
@@ -835,4 +838,233 @@ describe("Stamp Editor", () => {
...
+        await page.click(buttonSelector);
+        await page.waitForSelector("#newAltTextDialog", { visible: true });
+
+        // Check that the dialog has the correct title: "Edit..."
+        await page.waitForFunction(
+          "document.getElementById('newAltTextTitle').textContent.startsWith('Edit')"
+        );
+
+        // Click on create automatically toggle button.
+        await page.click("#newAltTextCreateAutomaticallyButton");
```
[the remark this answers is not in the document]
Comment (calixteman) [reply test/integration/stamp_editor_spec.mjs:987]: Yep it's something I'll test later.

**Review by timvandermeij — APPROVED** [review]
r=me, with a follow-up for the "Create automatically" toggle test, and preferably one to find out why always waiting for the input even in `clearInput` causes test failures since AFAICT that should have been safe (but the current solution is good now). Thank you for implementing test coverage for this flow!

**Pull request merged**